### PR TITLE
Unblock a patch release without the `cryptography` version upper bound

### PR DESCRIPTION
**Pull request by jessemyers**
-  Enable CircleCI builds
 -  Enable tox

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# CircleCI Configuration file
+
+machine:
+  python:
+      version: 3.6.2
```
Comment (jessemyers) [inline circle.yml]: I'm not trying for Python version parity, only what we currently deploy.

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    commands:
+      - echo "[distutils]" > ~/.pypirc
+      - echo "index-servers =" >> ~/.pypirc
+      - echo "    pypi" >> ~/.pypirc
+      - echo >> ~/.pypirc
+      - echo "[pypi]" >> ~/.pypirc
+      - echo "username:$PYPI_USERNAME" >> ~/.pypirc
+      - echo "password:$PYPI_PASSWORD" >> ~/.pypirc
+      - echo >> ~/.pypirc
+      - python setup.py register -r pypi
```
Comment (moshez) [inline circle.yml]: Register is deprecated. You can just upload nowadays.

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      - python setup.py sdist upload -r pypi
```
Comment (moshez) [inline circle.yml]: We should probably also build a wheel.
[In general, I prefer to upload with twine, but `setup.py upload` works ok nowadays]

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist = py36, lint
+
+[testenv]
+commands =
+    python setup.py nosetests --with-coverage --cover-package=openapi --cover-erase --cover-html
+    python setup.py sdist
```
Comment (moshez) [inline tox.ini]: I think tox automatically creates an sdist?

**Pull request merged**